ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Breakdown.kt

```diff
@@ -52,8 +52,10 @@ fun RBuilder.containerList(containers: List<FileContainer>, sizeType: Measurable
 @Suppress("UNUSED_PARAMETER")
 fun RBuilder.containerListItem(id: Int, container: FileContainer, sizeType: Measurable.SizeType, @RKey key: String) {
     div(classes = "accordion-item") {
-        containerListItemHeader(id, container, sizeType)
-        containerListItemBody(id, container, sizeType)
+        container.files?.let {
+            containerListItemHeader(id, container, sizeType)
+            containerListItemBody(id, container, sizeType)
+        } ?: containerWithoutFilesListItemHeader(container, sizeType)
```

Comment:
I think we can make this a lot simpler. We should be able to just disable the entry if there are no files in the container. That way we don't need to have 2 separate code paths. UI will show an expand hint, but it'll be disabled, which should be fine.